### ISSUE TITLE
Merge into Spring Security upstream

### ISSUE THREAD

**Issue by ynojima** [webauthn4j/webauthn4j-spring-security#11]
Merge this spring-security-webauthn module to the Spring Security upstream.

Pull request: https://github.com/spring-projects/spring-security/pull/6842

**Comment by ynojima** [comment]
This repository directly exposes the WebAuthn4J classes so that users can take full advantage of WebAuthn4J's functionality, but the style won't fit as Spring Security subprojects because they aim to abstract underlying library details.
Therefore, Efforts aimed at Spring Security subprojects and those as WebAuthn4J subprojects will be carried out in parallel. This repository will continue as WebAuthn4J subproject.
